CMakeLists.txt

```diff
@@ -311,6 +311,7 @@ option(USE_SYSTEM_ICU "Use the system-provided libicu. May fix startup crashes w
 
 option(USE_VALGRIND "Build Bun with Valgrind support (Linux only)" OFF)
 
+option(ZIG_COMPILER "Path to the Zig compiler" "system")
```

Comment:
does this break local development environments?

note that this file has the following later:

```
if(ZIG_COMPILER)
    if(ZIG_COMPILER STREQUAL "system")
        ...
    endif()
    ...
elseif(...)
    // auto-install zig compiler
endif()
```